adyenofficial.php

```diff
@@ -1118,7 +1118,7 @@ public function hookPaymentOptions()
                 }
 
                 $localPaymentMethod = new PrestaShop\PrestaShop\Core\Payment\PaymentOption();
-                $localPaymentMethod->setCallToActionText($this->l('Pay by ' . $paymentMethod['name']))
+                $localPaymentMethod->setCallToActionText(sprintf($this->l('Pay by %s'), $paymentMethod['name']))
```

Comment:
can you pls also add a suggestion to docs to let our merchants know how to use the %s in their translations? Is there any validation that we need to do before using $this->l('Pay by %s') in  sprintf() ? What happens if they put an extra %s in the translation like Pay by %s %s ?

Reply:
Docs suggestion: https://docs.adyen.com/plugins/prestashop#set-up-translations